py3status/modules/i3pystatus.py

```diff
@@ -0,0 +1,328 @@
+# -*- coding: utf-8 -*-
+"""
+Support i3pystatus modules in py3status.
+
+[i3pystatus](https://github.com/enkore/i3pystatus) is an alternative to
+py3status and provides a variety of modules.  This py3status module allows
+these modules to run and be display inside py3status.
```

Comment:
(Suggestion) `be displayed`
(Question)  Do we want `[i3py3status](https://...)` inside modules? First time here(?).
(Example). Maybe we should add `Examples:` before the descriptions too.... and have the descriptions inside the examples? For consistency, I guess... meaning `configuration parameters` should be the first thing to see, not the examples.

Reply:
I think having nice links is good - maybe this isn't the best format.  I'll have a think.  There is a precedent `i3block` but anyhow I want to look at the whole module docs issue.